src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -291,6 +313,107 @@ static TryCatchStatement composeUnaryRpcMethodSampleCode(
         .build();
   }
 
+  @VisibleForTesting
+  static TryCatchStatement composeUnaryPagedRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
```

Comment:
Nit: Consider using the order `method, arguments, clientType, repeatedResponseType, resourceNames` or `method, clientType, repeatedResponseType, arguments, resourceNames` (prefer the second one). The first has better semantic locality, the second groups the collection types together so the call order is easier to remember.

Reply:
Gotcha, good point. Update the code

Reply:
Could you please check again? The parameter order seems to be the same as before.

Reply:
Updated.